frontend-app/src/TemperatureControl.js

```diff
@@ -1,19 +1,25 @@
-import { useEffect, useState } from "react";
+import { useEffect, useState, useContext } from "react";
+import NotificationContext from "./context/NotificationContext";
 
 
-function TemperatureControl() {
-    const [temp, setTemp] = useState(10);
+function TemperatureControl({save, load, initial = 10}) {
+
+    const [temp, setTemp] = useState(parseInt(load() ?? initial));
+    const context = useContext(NotificationContext);
 
     useEffect(() => {
         document.title = `Температура ${temp} градусов`
+        save(temp);
     });
 
     const increase = () => {
         setTemp(temp >= 30 ? 30 : temp + 1);
+        context.increment('You`ve added 1 degree!');
     }
```

Comment:
получается что если у тебя счетчик будет 30
и при нажатии на кнопку, нотификация отобразится
```suggestion
        if (temp >= 30) {
                setTemp(temp >= 30 ? 30 : temp + 1);
                context.increment('You`ve added 1 degree!');
        }
    }
```

в decrease тоже самое